docs/sdks/flutter-sdk/overview.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Overview
+sidebar_position: 0
+---
+
+import SampleAppFlutter from '/docs/workflows/_sdk-setup/_sample-app-flutter.mdx';
+import InstallationFlutter from '/docs/workflows/_sdk-setup/_installation-flutter.mdx';
+import SetupFlutter from '/docs/workflows/_sdk-setup/_setup-flutter.mdx';
+import BindingAPasskeyFlutter from '/docs/workflows/_sdk-setup/_binding-a-passkey-flutter.mdx';
+import AuthenticationFlutter from '/docs/workflows/_sdk-setup/_authentication-flutter.mdx';
+import BindPasskeyUrlValidationFlutter from '/docs/workflows/_sdk-setup/_url-validation/_bind-passkey-url-validation-flutter.mdx';
+import AuthenticateUrlValidationFlutter from '/docs/workflows/_sdk-setup/_url-validation/_authenticate-url-validation-flutter.mdx';
+import ListingPasskeysFlutter from '/docs/workflows/_sdk-setup/_passkey-management/_listing-passkeys-flutter.mdx';
+import DeletingAPasskeyFlutter from '/docs/workflows/_sdk-setup/_passkey-management/_deleting-a-passkey-flutter.mdx';
```

Comment:
Oh I see, you're importing everything over (in response to the question in my bind PR). It might be easier to maintain if we kept just the one SDK Setup file?

Reply:
I agree if we're ditching these overview files now. This makes it easier to maintain both.

Reply:
Do you want to go ahead and rejoin these so we have one file for now?